src/main/java/com/algosenpai/app/controller/SceneController.java

```diff
@@ -34,6 +37,9 @@
 
 public class SceneController extends Application {
 
+    @FXML
```

Comment:
Do you really need to put this FXML in the Scene Controller?

Reply:
oh ya dont need coz the fxml file is different